### DISCUSSION TITLE
nextras/orm-intellij support for nextras/orm 4

### DISCUSSION THREAD

**Discussion by mrceperka** [nextras/orm#474] [Q&A] [answered]
Hi :wave:,
this question/issue mainly affects phpstorm plugin. Not the library itself.

:warning: **Problem**
I'm having trouble with `Cmd + Click` or suggestions in `findBy` and similar methods.

:eyes: **Observations**
Suggestions are working with `this->` prefix which is now deprecated.
They do not work without `this->` prefix.

With `this->`
![image](https://user-images.githubusercontent.com/22822651/101546567-44b68280-39a9-11eb-9a6c-dd61af01dd44.png)

Without `this->`
![image](https://user-images.githubusercontent.com/22822651/101546627-5dbf3380-39a9-11eb-9167-fdde68a05d99.png)

:computer: **Env**
Plugin version 0.7.1, phpstorm 2020.3RC, Linux

:question: **Question**
How do I fix this? Is this a known bug?

**Comment by hrach** [comment]
- Do you have actually installed Orm 4.0+?
- Could you please test suggestion in nextras/orm repository in test file BooksRepository::findByTags()? It works for me there.
- If it works for you there too, then I would need your project to check what's wrong directly debugging the plugin. 

Thanks for reporting!

Note: you may have open these issues and other suggestions on [the repository](https://github.com/nextras/orm-intellij/issues).

**Reply by mrceperka** [reply]
> Do you have actually installed Orm 4.0+

Yes, I am upgrading from 3.0

> Could you please test suggestion in nextras/orm repository in test file BooksRepository::findByTags()? It works for me there.

TODO

> If it works for you there too, then I would need your project to check what's wrong directly debugging the plugin

TODO

> Note: you may have open these issues and other suggestions

You are right, sorry, I've seen link to discussions on slack and I've just clicked :facepalm:

**Reply by mrceperka** [reply]
Fixed by `Invalidate Caches`

**Reply by hrach** [reply]
WOW! Didn't expect this outcome! Thanks for resolving it!

**Comment by mrceperka** [comment, marked as answer]
Fixed by `Invalidate Caches`
